good-enough-practices-for-scientific-computing.bib

```diff
@@ -6,6 +6,12 @@ @misc{brown2015
 	url = {http://ivory.idyll.org/blog/2015-growing\--sustainable\--software\--development-process.html}
 }
 
+@misc{ffIllinois,
```

Comment:
I removed a dedicated sentence about this resource and included it as a proper citation within a sentence that has the same sprit.